src/e3/anod/spec.py

```diff
@@ -274,6 +274,25 @@ def enable_name_generator(self) -> bool:
         """
         return False
 
+    @property
+    def readme_filename(self) -> str:
+        """The readme filename to use.
+
+        :return: The readme filename to use.
+            Note that the return value cannot be None or an empty string
+        """
+        return f"README-{self.component}"
+
+    @property
+    def readme_ext(self) -> str:
+        """The readme extension to use.
+
+        :return: The extension used by the readme file.
+            Note that if this return value begin by dot(s), these dots will be
+            discarded (Example: ..txt = .txt = txt)
```

Comment:
Seeing the method body, I would say that the documentation is wrong :)
Either say _the returned value should be stripped of its starting dots._, or change how the property is used.

Reply:
Currently, this comment isn't true, but it should become true soon. And it's almost certain that I'll forget to update this docstring, so I did it a little early 😄 

Do you think this is really a problem?

Reply:
We've completely changed the way the code works, so this comment is now obsolete. I'll take the liberty of resolving it.